index.html

```diff
@@ -169,25 +168,27 @@ <h3>
       </section>
       <section class="informative">
         <h3>
-          Don't add redundant roles
+          Avoid specifying redundant roles
         </h3>
         <p>
-          The following uses a `role=button` on a [^button^] element. This is
-          unnecessary, as "button" is already exposed as the implicit role for
-          the element. In practice this redundancy will likely not have any
-          unforeseen side effects, other than unnecessarily making the markup
-          more verbose, and incorrectly signaling to other authors that this
-          practice is useful.
+          The following example illustrates a [^button^] element which has also been
+          provided an explicit `role=button`. Specifying this role is unnecessary, as a "button" 
+          element is already exposed with an implicit `button` role. In practice this particular
+          instance of redundancy will likely not have unforeseen side effects, other than 
+          unnecessarily making the markup more verbose, and incorrectly signaling to other authors 
+          that this practice is useful. Please review the section ["Be cautious of side effects"](#side-effects)
+          for an example of where specifying unnecessary roles can be problematic.
         </p>
         <pre class="example HTML" title="Redundant role on button">
           &lt;!-- Avoid doing this! -->
           &lt;button role="button"&gt;...&lt;/button&gt;
         </pre>
         <p>
           Similarly, the following uses a `role=group` on a [^fieldset^] element, and a `role=Main` on a [^main^] element.
```

Comment:
lowercase the "M" in `role=Main`

Reply:
or is this done on purpose?

if not, also correct it in the code sample that follows

Reply:
later in that paragraph it references the case sensitivity section.

Reply:
feels like it's trying to make two separate points though, oddly mashed together.  it starts off with "these two have roles, but they're redundant because in practice it's the same as long as they're lowercase". but one isn't. but it then doesn't explain what the deal with that second one not being lowercase is.

best to keep the two points separate, or explain them properly.

Reply:
this wasn't introduced with this PR, but rather https://github.com/w3c/html-aria/pull/333 which you approved :P

Reply:
i'm bustin on you :)

you want to remove or revise it?

Reply:
"bustin' makes me feel good"

nah it's ok, will revisit that in another PR, as commented. this PR per se is fine